### DISCUSSION TITLE
#general

### DISCUSSION THREAD

**Discussion by Avdhesh-Varshney** [code-a2z/jarvis#133] [General]
### 💭 This channel is for general discussion on this project.

**Comment by Avdhesh-Varshney** [comment]
@kavya410004 I don't know any such api for TV serials. I think you can review TMDB

**Reply by kavya410004** [reply]
> @kavya410004 I don't know any such api for TV serials. I think you can review TMDB

That will help. Thank you. Can you assign issue #48 to me?

**Comment by Avdhesh-Varshney** [comment]
@AnushkaS77 You can review another gif `image.gif` present in assets folder

**Reply by AnushkaS77** [reply]
yes i did but the error still persists

**Reply by Avdhesh-Varshney** [reply]
Ok, no problem. You just do the required changes in your pr. Afterwards, I'll edit your pr. 
Convert video to gif and keep in `assets` folder. Rest are mentioned in pr
